packages/core/src/resolver.js

```diff
@@ -27,9 +46,11 @@ export default class Resolver {
     this._resolve = require.resolve
   }
 
-  resolveModule (path) {
+  resolveModule (path, { requirePath } = {}) {
     try {
-      return this._resolve(path, {
+      const pathResolve = requirePath ? createRequire(requirePath).resolve : this._resolve
```

Comment:
What do you think to cache `createRequire` (per `requirePath`)? (maybe doing it directly with `createRequire` util or `getRequire(requiePath)`?)

Reply:
🤔 I don't think it would have a noticeable effect on performance. The function with the highest complexity seems to be [`Module._nodeModulePaths`](https://github.com/nodejs/node/blob/c17dcb32533aa007dfbf507d22c28ef3c7c11c29/lib/internal/modules/cjs/loader.js#L1241), but I don't really think it would affect the performance.